src/classes/WalletManager/index.ts

```diff
@@ -401,4 +405,18 @@ export class WalletManager {
     private _addEventListener<T>(event: string, callback: (response: T) => void) {
         return this._nativeProvider?.on?.(event, response => callback.bind(this)(response))
     }
+
+    /**
+     * Retrieves the Wallet connector by his ID.
+     *
+     * @returns {WalletConnector} - Returns the Wallet connector listed inside connectors array.
+     */
+    private _getConnector(connectorId: string): WalletConnector {
+        const connector = this._connectors.find(connector => connector.id === connectorId)
```

Comment:
what do you think about mapping connectors to their ids?

Reply:
I like it :) Also, the code will look much cleaner :ok: